images/virtualization-artifact/pkg/controller/vm_reconciler_test.go

```diff
@@ -374,6 +392,7 @@ var _ = Describe("Apply VM changes", func() {
 		nsName := "test-ns-2"
 		vmName := "test-vm-2"
 		vmipName := "test-vmip"
+		vmcpuName := "test-vmip"
```

Comment:
```suggestion
		vmcpuName := "test-vmcpu"
```
Just for consistency =)